src/main/java/mate/academy/spring/service/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.dto.request.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.response.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```

Comment:
```suggestion
    public CinemaHallResponseDto toResponseDto(CinemaHall cinemaHall) {
```
better to call it more explicetily